sdk/rust/src/http.rs

```diff
@@ -705,6 +716,21 @@ impl std::fmt::Display for streams::Error {
 
 impl std::error::Error for streams::Error {}
 
+struct DebugBody<'a>(&'a [u8]);
```

Comment:
I know I suggested this, and we're already debug printing headers, but are we worried about security implications of this? This can easily leak sensitive information into logs.

Reply:
:shrug: I'm ambivalent on this one. As an appeal to convention, `http::Request` appears to  dump everything: https://docs.rs/http/0.2.9/src/http/request.rs.html#704-715